src/MediaPlayer/MPNowPlayingInfoCenter.cs

```diff
@@ -115,6 +123,11 @@ internal NSDictionary ToDictionary ()
 			return dict;
 		}
 
+		internal void Add (NSMutableDictionary dictionary, NSObject key, NSObject value) {
```

Comment:
style for `{` (on next line for methods)
it also looks like it only needs to be `private` (default) and not internal